src/cordial_sound/scripts/wav_file_publisher.py

```diff
@@ -18,21 +22,33 @@ def __init__(self):
             'cordial/sound/wav/header_length',
             24
         )
-        rospy.Subscriber(rospy.get_param('cordial_sound/play_wav_topic'), String, self.play_wav_file)
-        self._sound_publisher = rospy.Publisher(rospy.get_param('cordial_sound/play_stream_topic'), Sound, queue_size=1)
+        rospy.Subscriber(rospy.get_param(
+            'cordial_sound/play_wav_topic'), String, self.play_wav_file)
+        self._sound_publisher = rospy.Publisher(rospy.get_param(
+            'cordial_sound/play_stream_topic'), Sound, queue_size=10)
```

Comment:
Is there any reason to have the queue size of 10? In general, I prefer to use a queue size of 1, so that we don't get delayed.